action.yaml

```diff
@@ -37,6 +37,11 @@ inputs:
     required: false
     default: false
 
+  default_token:
```

Comment:
Should we name this something like `experimental_*` or `internal_*` so that developers know (without needing to read the description) that we are using it for internal purposes?

Reply:
good idea. Will rename to `internal_`